Vostok.Metrics.System/Host/DiskSpaceCollector.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Vostok.Metrics.System.Helpers;
+
+namespace Vostok.Metrics.System.Host
+{
+    internal class DiskSpaceCollector
+    {
+        public void Collect(HostMetrics metrics)
+        {
+            metrics.DiskSpaceInfos = GetDiskSpaceInfos().ToDictionary(x => x.Name);
+        }
+
+        private IEnumerable<DiskSpaceInfo> GetDiskSpaceInfos()
+        {
+            foreach (var drive in DriveInfo.GetDrives().Where(x => x.IsReady && x.DriveType == DriveType.Fixed))
+            {
+                var result = new DiskSpaceInfo();
+
+                try
+                {
+                    result.Name = drive.Name.Replace(":\\", string.Empty);
+                    result.FreeBytes = drive.TotalFreeSpace;
+                    result.TotalCapacityBytes = drive.TotalSize;
+                    result.FreePercent = drive.TotalFreeSpace * 100d / drive.TotalSize;
```

Comment:
1. Reuse filled properties instead of querying the drive all over again.
2. Handle zero total size (just in case).